libs/wingsdk/src/platform/parameter-registrar.ts

```diff
@@ -41,12 +41,27 @@ export class ParameterRegistrar extends Construct {
     return this.parameterValueByPath[path];
   }
 
+  /**
+   * Return all parameters as Json
+   *
+   * @returns the schema as a string
+   */
+  public json(): Json {
+    return this._rawParameters as Json;
+  }
```

Comment:
I don't really have an alternative solution, but we have to watch out with using `std` types outside of the `std` namespace. This `Json` class isn't really real since much of its behavior is technically created via macros. So if a consumer of the SDK outside of wing calls this function and attempts to call any of the macro functions they'll get an error.

Maybe that's okay for the platform namespace for now though.

Reply:
Yes that's a good call. The typescript experience will not be nice.

For most std types there's a typescript equivalent so we just use these but we don't have for Json, ha?

I am wondering if we we should just use `any` for this and maybe annotate it somehow for Wing to know it's a `json`.